docs/guide/gwt-integration/client-bundles-and-styles.md

```diff
@@ -47,7 +47,7 @@ We can then simply access our bundle instance from the template:
 You could also directly expose your ImageResource to the template, it's really up to you.
 
 ::: tip
-You can use [Component inheritance](../composition/extending-components) to declare a base component in your app and avoid
+You can use [Component inheritance](../essentials/components.html) to declare a base component in your app and avoid
```

Comment:
```suggestion
You can use [Component inheritance](../essentials/components) to declare a base component in your app and avoid
```

No need for .html, we are using VuePress: https://vuepress.vuejs.org/, so the link is actually to the md file :).